plans/cmb3776.md

```diff
@@ -0,0 +1,19 @@
+## Status
+
+I am trying to recreate flick issue on iOS simulator using code provided in description.
+
+## Goal
+
+* I will resolve [issue #345](https://github.com/ios-driver/ios-driver/issues/345): flick is not working on iOS simulator.
+
+* I will implement automation for new multi-touch gestures.
+
+## Plan
+
+1. Recreate issue using code provided in description.
+1. Identify and resolve flick issue.
+1. Submit pull request and close issue.
```

Comment:
it's not clear to me whether this is in your plan, but you should write a test case that reproduces the problem and include that in your PR.